### ISSUE TITLE
Fetal pericardial effusion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7223]
**Preferred term label:**
Fetal pericardial effusion

**Synonyms**

**Definition (free text, please give PubMed ID)**
todo -- find reference for thickness of PE, is this gestational age dependent?

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Fetal hemopericardium
https://obgyn.onlinelibrary.wiley.com/doi/full/10.1002/uog.16847

**Comment by patt1morgan** [comment]
@pnrobinson 
Definition: Fetal Pericardial effusion ,an accumulation of fluid in the chest cavity of a developing fetus, is defined on ultrasound examination when the heart is partially or completely surrounded by fluid that is seen in all views, is usually present around the atrioventricular groove, and the larger region is >2 mm . If it measures <4 mm it is considered small and if it measures >4 mm it is considered large. In cases of massive pericardial effusion, a posterior displacement of the lungs can be found on the four-chamber view (https://www.emjreviews.com/interventional-cardiology/article/fetal-pericardiocentesis/)- also includes images.

Comment : Fetal pericardial effusion appears in  pathologies such as hydrops fetalis, heart structural or rhythm alterations, however, it can also be observed in isolation. An increase in its incidence has been observed in relation to the presence of severe pathologies. PMID: 33781717

Parent term: Pleural effusion HP:0002202

**Comment by pnrobinson** [comment]
Def: An abnormal accumulation of fluid in which the heart is partially or completely surrounded by fluid that is seen in all views and the thickness of the fluid as observed by prenatal ultrasound is above age-dependent norms. 

Comment : Fetal pericardial effusion appears in conditions such as hydrops fetalis, heart structural or rhythm alterations, however, it can also be observed in isolation 
PMID:33781717

(agreed)
